code/game/machinery/computer/guestpass.dm

```diff
@@ -3,7 +3,7 @@
 /////////////////////////////////////////////
 /obj/item/card/id/guest
 	name = "guest pass"
-	desc = "Allows temporary access to station areas."
+	desc = "Allows temporary access to ship areas."
```

Comment:
would be better if it did not mention ship nor station at all, so it can be used better outside of horizon